tests/conftest.py

```diff
@@ -7,6 +9,8 @@
 from tests.test_magic_link.magic_link_data import expected_magic_link_data
 from tests.test_magic_link.magic_link_data import expected_magic_link_response
 
+multiprocessing.set_start_method("fork")  # Required on macOSX
```

Comment:
🔧 Add check that user is on MacOS, otherwise the windows users (whose lives are already terrible from using windows) will have tests crash on them.

Reply:
I think this should be another ticket as we have it on a number of repos - and we should probably make a solution that doesn't require os specific code if possible

Reply:
I have temporarily added the check 🤔

Reply:
But looks like Ram's got it covered, thanks Ram!